### ISSUE TITLE
Remove Warnings

### ISSUE THREAD

**Issue by ferredel** [akserg/ng2-toasty#9]
As requeriment in Beta 17....

``````
Template parse warnings:
"#" inside of expressions is deprecated. Use "let" instead! ("
    <div id="toasty" [ngClass]="[position]">
        <ng2-toast [ERROR ->]*ngFor="#toast of toasts" [toast]="toast" (closeToast)="closeToast(toast)"></ng2-toast>
    </div>"): Toasty@2:19```
``````

**Comment by akserg** [comment]
Thak you to remind me to migrate to the latest version of Angular.

**Comment by akserg** [comment]
Fixed in v 1.6
